### PR TITLE
Update pl.js (v5.7.0)

### PR DESCRIPTION
**Pull request by ectryliz**
I left some of the translation untouched because there are issues with grammar when templating, I also added myself to code owners.

### CODE REVIEW

**Review by xhg4 — CHANGES_REQUESTED** [review]
All issues are addressed in the comments. As usual feel free to change the translation to something that's also different from what I wrote in a comment if you think there's a better translation
Apart from the small issues, this is one of the best translations I've seen while working on pl.js, it's great to be able to collaborate with such a good translator

File: pl.js [hunk 1]
```diff
@@ -750,8 +750,8 @@ server.config.autoJump Automatyczne skakanie
 server.config.bDrop Upadek pocisków
 server.config.thirdPerson Tryb trzecioosobowy
 server.config.requireCaptcha Wymagaj captchy
-server.config.nameTags Ukryj nazwy nad graczami
-server.config.nameTagsFR Hide Nametags (Allies)
+server.config.nameTags Ukryj nazwy przeciwników
```
[the remark this answers is not in the document]
Comment (ectryliz) [reply pl.js:753]: The original translation was changed to only opponents, the github doesn't show that change. https://github.com/mty22/krunker.io-translations/blob/66c9cd44b5074ef7e4917741358ccce5008bedf7/en.js#L753

File: pl.js [hunk 2]
```diff
@@ -1006,8 +1006,8 @@ server.message.kicked {0} został wyrzucony
...
 server.message.votekicked {0} został wyrzucony przez głosowanie
 server.message.arrested {0} został aresztowany przez K.P.D.
-event.easter.golden.spawn A Golden egg is near
-event.easter.golden.despawn A Golden egg has gone
+event.easter.golden.spawn Złote jajo jest blisko
+event.easter.golden.despawn Złote jajko zniknęło
 event.easter.found.new {0} found {1}/{2} Eggs
-event.easter.found.old You already discovered this egg!
+event.easter.found.old Odkryłeś już to jajo!
 event.easter.completed {0} completed the Egg hunt!
```
[the remark this answers is not in the document]
Comment (ectryliz) [reply pl.js:1013]: Still would be strange, "You odnalazł wszystkie jaja!" where it should be "Odnalazłeś wszystkie jaja!"

[the remark this answers is not in the document]
Comment (xhg4) [reply pl.js:1013]: Oh right, sorry, I completely forgot how it renders if you're the one mentioned in {0}

**Review by xhg4 — DISMISSED** [review]
Perfect, ready to merge @mty22

File: pl.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (xhg4) [reply pl.js:753]: Oh, okay, I see

**Comment by ectryliz** [comment]
@xhg4 I added some small changes to other translations, would appreciate if you could review it again.

**Comment by xhg4** [comment]
Okay, I've just checked the commit "update pl.js" and everything looks fine, ready to merge

The commit "merge branch ' mty22:main ' into patch-1"... I guess it's better if @mty22 takes a look at it as:
1. I have no idea what's going on in the majority of the commit
2. That commit should probably be in a separate pull request anyways

**Comment by ectryliz** [comment]
It's upstream fetch in my own repository, it does absolutely nothing here but github decided to show it (oh well).

**Comment by xhg4** [comment]
Ah, I see
Just github being github ig

**Review by xhg4 — APPROVED** [review]

**Pull request merged**